instagram-page-react/src/pages/profilePage/reducers/actionTypes/profilePage.posts.js

```diff
@@ -0,0 +1,7 @@
+const SET_POST = 'setPost';
```

Comment:
Constant names and values are both in capitals
setPost <> SET_POST

Reply:
SET_POST <> SAVE_POST